pkg/processor/runtime/runtime.go

```diff
@@ -63,17 +64,21 @@ type Runtime interface {
 
 	// Terminate sends a signal to the runtime process and waits for it to exit
 	Terminate() error
+
+	// GetControlMessageBroker returns the control message broker
+	GetControlMessageBroker() controlcommunication.ControlMessageBroker
 }
 
 // AbstractRuntime is the base for all runtimes
 type AbstractRuntime struct {
-	Logger         logger.Logger
-	FunctionLogger logger.Logger
-	Context        *nuclio.Context
-	Statistics     Statistics
-	databindings   map[string]databinding.DataBinding
-	configuration  *Configuration
-	status         status.Status
+	Logger               logger.Logger
+	FunctionLogger       logger.Logger
+	Context              *nuclio.Context
+	Statistics           Statistics
+	ControlMessageBroker controlcommunication.ControlMessageBroker
```

Comment:
if you expose it via a `Get...` does it need to be a public property of runtime struct?

Reply:
Well, if we initialize it AFTER creating a new abstractRuntime (Like we do here: https://github.com/nuclio/nuclio/blob/e41ed17ed67df0bc478ae6720a3cfabb5434b8dc/pkg/processor/runtime/rpc/abstract.go#L322-L323) than it needs to be public.

However, making it public doesn't let the worker access it, hence I created the Getter (used here: https://github.com/TomerShor/nuclio/blob/f644f71bc5a0c8e027162fa16282c5b18656cc5a/pkg/processor/worker/worker.go#L155-L156)